src/@ionic-native/plugins/flurry-analytics/index.ts

```diff
@@ -0,0 +1,212 @@
+import { Plugin, Cordova } from '@ionic-native/core';
+import { Injectable } from '@angular/core';
+
+export interface FlurryAnalyticsOptions {
+  /** Flurry API key is required */
+  appKey?: string;
+  /**
+   * Optional parameters
+   */
+  /** Overrides the version of the app */
+  version?: string;
+  /**
+   * How long can the app be paused before a new session is created,
+   * must be less than or equal to five for Android devices
+   */
+  continueSessionSeconds?: number;
+  /**
+   * Set id of the user
+   */
+  userId?: string;
+  /**
+   * Set gender of the user
+   * Valid values are "m", "M", "f" and "F"
+   */
+  gender?: string;
+  /**
+   * Set age of the user
+   */
+  age?: number;
+  /**
+   * Set error for log
+   * Values: VERBOSE, DEBUG, INFO, WARN, ERROR
+   */
+  logLevel?: string;
+  /**
+   * Defaults to false
+   */
+  enableLogging?: boolean;
+  /**
+   * Should every event show up the app's log, defaults to true
+   */
+  enableEventLogging?: boolean;
+  /**
+   * Should app crashes be recorded in flurry, defaults to false, iOS only
+   */
+  enableCrashReporting?: boolean;
+  /**
+   * Should the session continue when the app is the background, defaults to false, iOS only
+   */
+  enableBackgroundSessions?: boolean;
+  /**
+   * Should data be pushed to flurry when the app closes, defaults to true, iOS only
+   */
+  reportSessionsOnClose?: boolean;
+  /**
+   * Should data be pushed to flurry when the app is paused, defaults to true, iOS only
+   */
+  reportSessionsOnPause?: boolean;
+}
+
+export interface FlurryAnalyticsLocation {
+  latitude: number;
+  longitude: number;
+  verticalAccuracy?: number; // optional iOS only
```

Comment:
The `optional iOS only` documentation should be stated above the parameter (preferably along with some description). Just like the way you have it in the interface above.